mayastor/src/bdev/nexus/nexus_child.rs

```diff
@@ -489,7 +494,13 @@ impl NexusChild {
     pub(crate) async fn destroy(&self) -> Result<(), NexusBdevError> {
         if self.device.is_some() {
             self.set_state(ChildState::Destroying);
-            device_destroy(&self.name).await
+            info!("{} destroying underlaying block device", self.name);
```

Comment:
should be "underlying" rather than underlaying

Reply:
Indeed ! Thanks for highlighting the typo.